packages/serverpod_flutter/lib/src/flutter_connectivity_monitor.dart

```diff
@@ -18,17 +18,19 @@ class FlutterConnectivityMonitor extends ConnectivityMonitor {
     // Start listening to connection status changes.
     _connectivitySubscription =
         Connectivity().onConnectivityChanged.listen((event) {
+      final bool noConnectivity =
+          event.length == 1 && event.first == ConnectivityResult.none;
+
```

Comment:
Question: Are we guaranteed that this will be of length 1 if the ConnectivityResult is none?

We could also implement this like this to guard against different lengths
```dart
final bool noConnectivity = event.any((connectivity) => connectivity == ConnectivityResult.none);
```
Suggestion:
I also think we should rename this variable. I rather have a variable called `hasConnectivity` or `isConnected` where the name is not negated. This makes it easier to read / understand if you negate the variable later.

`!noConnectivity` meaning "not no connectivity" which is more cumbersome than it has too be.